e2e/common/getAccessTokenHelper.ts

```diff
@@ -0,0 +1,63 @@
+import { APIRequestContext } from "@playwright/test";
+import * as dotenv from "dotenv";
+
+dotenv.config();
+
+const citizenCreateUserData = {
+  grant_type: "client_credentials",
+  client_id: "prl-cos-api",
+  client_secret: process.env.IDAM_SECRET,
+  scope: "profile roles",
+};
+
+const daCourtNavCreateCaseData = {
+  grant_type: "password",
+  client_id: "courtnav-service",
+  client_secret: process.env.COURTNAV_SECRET,
+  scope: "openid profile roles",
+  username: process.env.COURTNAV_USERNAME,
+  password: process.env.COURTNAV_PASSWORD,
+};
+
+/**
+ * Function to get an access token from the IDAM service
+ * @param {APIRequestContext} apiContext The API request context
+ * @returns {Promise<string>} The access token if successful, otherwise throws an error
+ */
+export async function getAccessToken(
+  option: string,
+  apiContext: APIRequestContext,
+): Promise<string> {
+  try {
+    let data;
+    if (option === "citizenCreateUser") {
+      data = citizenCreateUserData;
+    } else if (option === "daCourtNavCreateCase") {
+      data = daCourtNavCreateCaseData;
```

Comment:
Do we only expect to keep this 2 options or could more be needed? Might want to consider something more scalable if we are needing to increase the amount of options, otherwise think this is fine

Reply:
i can change it to a switch function and that should work with more options should we need them - will need for any other case creation posts we do